demo/agilebank/templates/k8srequiredprobes_template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
```

Comment:
Any chance we could add some unit tests and put this in the `library` folder?

Reply:
How can I run unit tests locally?
When I execute (including existing unit tests in the library) `opa test src_test.rego`,  I get an error: `rego_unsafe_var_error: var violation is unsafe`.

Reply:
Nm. It seems I was missing the rego rules with violation:
`opa test src.rego src_test.rego`
`PASS: 1/1`